docset/winserver2022-ps/pki/Export-Certificate.md

```diff
@@ -11,66 +11,78 @@ title: Export-Certificate
 # Export-Certificate
 
 ## SYNOPSIS
+
 Exports a certificate from a certificate store into a file.
 
 ## SYNTAX
 
 ```
-Export-Certificate [-Type <CertType>] [-NoClobber] [-Force] -FilePath <String> -Cert <Certificate> [-WhatIf]
- [-Confirm] [<CommonParameters>]
+Export-Certificate [-Type <CertType>] [-NoClobber] [-Force] -FilePath <String>
+ -Cert <Certificate> [-WhatIf] [-Confirm] [<CommonParameters>]
 ```
 
 ## DESCRIPTION
-The **Export-Certificate** cmdlet exports a certificate from a certificate store to a file.
-The private key is not included in the export.
-If more than one certificate is being exported, then the default file format is SST.
-Otherwise, the default format is CERT.
-Use the **Type** parameter to change the file format.
+
+The `Export-Certificate` cmdlet exports a certificate from a certificate store to a file. The
+private key is not included in the export. If more than one certificate is being exported, then the
+default file format is `SST`. Otherwise, the default format is `CERT`. Use the **Type** parameter to
+change the file format.
 
 ## EXAMPLES
 
 ### EXAMPLE 1
-```
-PS C:\>$cert = Get-ChildItem -Path cert:\CurrentUser\My\EEDEF61D4FF6EDBAAD538BB08CCAADDC3EE28FF
 
-PS C:\>Export-Certificate -Cert $cert -FilePath c:\certs\user.sst -Type SST
+```powershell
+$cert = Get-ChildItem -Path Cert:\CurrentUser\My\EEDEF61D4FF6EDBAAD538BB08CCAADDC3EE28FF
+
+Export-Certificate -Cert $cert -FilePath C:\Certs\user.sst -Type SST
 ```
 
-This example exports a certificate to the file system as a Microsoft serialized certificate store without its private key.
+This example exports a certificate to the file system as a Microsoft serialized certificate store
+without its private key.
 
 ### EXAMPLE 2
-```
-PS C:\>$cert = Get-ChildItem -Path cert:\CurrentUser\My\EEDEF61D4FF6EDBAAD538BB08CCAADDC3EE28FF
 
-PS C:\>Export-Certificate -Cert $cert -FilePath c:\certs\user.cer
+```powershell
+$cert = Get-ChildItem -Path Cert:\CurrentUser\My\EEDEF61D4FF6EDBAAD538BB08CCAADDC3EE28FF
+
+Export-Certificate -Cert $cert -FilePath C:\Certs\user.cer
 ```
 
-This example exports a certificate to the file system as a DER-encoded `.cer` file without its private key.
+This example exports a certificate to the file system as a DER-encoded .cer file without its
```

Comment:
Because the file extension isn't a parameter value, and because I could not find specific style guidelines for file extensions in the PowerShell-Docs style guide, I removed the backticks to comply Microsoft style for extensions.

Reply:
@robderickson for PowerShell content, we do backtick file extensions.